docs/templates/eap-cd-tx-recovery-s2i.adoc

```diff
@@ -169,6 +194,15 @@ for more information.
 |jolokia | 8778 | `TCP`
 |http | 8080 | `TCP`
 |ping | 8888 | `TCP`
+.8+| `${APPLICATION_NAME}-amq`
+|jolokia | 8778 | `TCP`
```

Comment:
During this week testing I discovered that this has changed. Jolokia port for AMQ 7 is now 8161 AFAIK. See https://github.com/jboss-container-images/jboss-amq-7-broker-openshift-image/blob/amq71-dev/templates/amq-broker-71-basic.yaml#L27

Reply:
Nice catch, I'll fix and update the build shortly.